### PR TITLE
Reference generalized ontological model

### PR DESCRIPTION
**Pull request by MuhammadYahta**
RGOM is developed for semantically modeling the manufacturing production line. It has been used for integration the manufacturing production line data of football production lines producing balls for adidas.

### CODE REVIEW

**Comment by TallTed** [comment]
Please add to the README.md — the names, GitHub handles, and any other relevant contact details you choose (email addresses, phone numbers, website URLs, etc.) for the maintainers of these `/rgom/` redirections.

**Comment by MuhammadYahta** [comment]
Hi

Thank you for your response.  Can you please explain what is meant by the names, GitHub handles?

With kind regards, M. Yahya    On Friday, January 20, 2023 at 09:47:58 PM GMT, Ted Thibodeau Jr ***@***.***> wrote:  
 
 


Please add to the README.md — the names, GitHub handles, and any other relevant contact details you choose (email addresses, phone numbers, website URLs, etc.) for the maintainers of these /rgom/ redirections.

**Comment by MuhammadYahta** [comment]
> Please add to the README.md — the names, GitHub handles, and any other relevant contact details you choose (email addresses, phone numbers, website URLs, etc.) for the maintainers of these `/rgom/` redirections.

We will make available the website documentation when it become ready.

File: rgom/.htaccess [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Options +FollowSymLinks
+
+# Rewrite engine setup
+RewriteEngine On
+
+# Redirect to w3id.org/rgom/
+RewriteRule	^$ https://github.com/MuhammadYahta/rgom/blob/main/rgom.owl [R=333,L]
```
Comment (dgarijo) [inline rgom/.htaccess:7]: I think you mean a 303 redirection

**Comment by dgarijo** [comment]
Thanks, please have a look at my comment. 333 does not seem to be a valid redirection response code https://developer.mozilla.org/en-US/docs/Web/HTTP/Status

**Comment by MuhammadYahta** [comment]
@dgarijo I change it to 303, please check

**Pull request merged**